string type
float32 nstep
float32[] pose
float32 gain
---
bool success
string status_message
